tests/integration/test_dss.py

```diff
@@ -12,6 +12,27 @@
 NOTEBOOK_NAME = "test-nb"
 
 
+def test_status_before_initialize(cleanup_after_initialize) -> None:
```

Comment:
Do we need `cleanup_after_initialize` in every test case? At first, I concluded from that that in every test case we clean up and re-initialize but then I saw that it is not this way. If we need it, should we redesign how the clean up takes place? Should we have clean up as a separate test case (for `dss purge`) instead of a fixture?

Reply:
So lets keep the clen up as it is but then we will remove it in your purge command ?